Funcky/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,10 @@
 #nullable enable
+static Funcky.Functional.NoOperationAsync() -> System.Threading.Tasks.Task
```

Comment:
The change should be reflected in the changelog

Reply:
7c5bb1f8f5f7ecabc9b02b97cee995c3210be3dc